lib/rbs_rails/active_record.rb

```diff
@@ -166,6 +167,41 @@ def reload_#{a.name}: () -> #{type_optional}
         end.join("\n")
       end
 
+      private def generated_association_methods
+        # @type var sigs: Array[String]
+        sigs = []
+
+        # Needs to require "active_storage/engine"
+        if klass.respond_to?(:attachment_reflections)
+          sigs << "module GeneratedAssociationMethods"
```

Comment:
This is a tracing of the rails module.

https://github.com/rails/rails/blob/18707ab17fa492eb25ad2e8f9818a320dc20b823/activestorage/lib/active_storage/attached/model.rb#L53
https://github.com/rails/rails/blob/4ae0390c1051a39ec16a4a95451cd06172a1b260/activerecord/lib/active_record/core.rb#L368-L376